code/_globalvars/special_traits/traits.dm

```diff
@@ -144,18 +144,6 @@
 	character.mind.special_items["Dagger"] = /obj/item/weapon/knife/dagger
 	character.adjust_skillrank(/datum/skill/combat/knives, 2, TRUE)
 
-/datum/special_trait/cunning_linguist
-	name = "Cunning Linguist"
-	greet_text = span_notice("I know an extra language, and I can be quite seductive")
-	weight = 100
-
-/datum/special_trait/cunning_linguist/on_apply(mob/living/carbon/human/character, silent)
-	switch(rand(1,2))
-		if(1)
-			character.grant_language(/datum/language/elvish)
-		if(2)
-			character.grant_language(/datum/language/hellspeak)
-
```

Comment:
Why not just make ploygot rarer instead

Reply:
That's why special traits have can_apply

Reply:
Eitherway if you are adding more things to polygot make it rarer as well

Reply:
its really common for something that will give you the majority of known languages